Userland/Applications/PixelPaint/Tools/BrushTool.cpp

```diff
@@ -172,4 +175,21 @@ GUI::Widget* BrushTool::get_properties_widget()
     return m_properties_widget.ptr();
 }
 
+NonnullRefPtr<Gfx::Bitmap> BrushTool::build_cursor()
+{
+    // FIXME: changing the image scale does currently not trigger a cursor update :-/
+    //        only changes to the brush size will take account for the current scale setting
+    auto scale = m_editor ? m_editor->scale() : 1;
+    auto scaled_size = size() * scale;
+    auto containing_box_size = 2 * scaled_size;
+    NonnullRefPtr<Gfx::Bitmap> new_cursor = Gfx::Bitmap::try_create(Gfx::BitmapFormat::BGRA8888, Gfx::IntSize(containing_box_size, containing_box_size)).release_value_but_fixme_should_propagate_errors();
+
+    auto painter = Gfx::Painter(new_cursor);
+    painter.fill_rect(Gfx::IntRect(0, 0, containing_box_size, containing_box_size), Color::Transparent);
+    painter.draw_line({ scaled_size - 5, scaled_size }, { scaled_size + 5, scaled_size }, Color::MidGray, 2);
+    painter.draw_line({ scaled_size, scaled_size - 5 }, { scaled_size, scaled_size + 5 }, Color::MidGray, 2);
+    painter.draw_circle_arc_intersecting(Gfx::IntRect(0, 0, containing_box_size, containing_box_size), { scaled_size, scaled_size }, scaled_size - 1, Color::MidGray, 1);
```

Comment:
I don't think that `fill_rect(Color::Transparent)` is doing anything (bitmaps already start transparent and even if it were not `fill_rect` would alpha blend, you'd need to use `clear_rect`). Also maybe try anti-aliasing that circle :^)
```suggestion
        Gfx::Painter painter { target_bitmap };
        Gfx::AntiAliasingPainter aa_painter { painter };
        painter.draw_line({ scaled_size - 5, scaled_size }, { scaled_size + 5, scaled_size }, Color::MidGray, 2);
        painter.draw_line({ scaled_size, scaled_size - 5 }, { scaled_size, scaled_size + 5 }, Color::MidGray, 2);
        aa_painter.draw_ellipse(Gfx::IntRect(0, 0, containing_box_size, containing_box_size), Color::MidGray, 1);
```

Reply:
Hello @MacDue,
thanks for the review. I've done your recommended changes and the aa circle looks much better :-)
Regards,
Torsten